packages/playground/src/types/index.ts

```diff
@@ -12,12 +12,19 @@ export interface K8SWorker {
   planetary: boolean;
   rootFsSize: number;
   farm?: Farm;
+  selectedNode: any;
```

Comment:
can we set it as optional and set the type to number; I think it will need some changes in related weblets, but if that is possible will be better than using any.
we can declare a new interface for `selectedNode` if that is needed

Reply:
done.